api_app/views.py

```diff
@@ -106,28 +121,55 @@ def _multi_analysis_request(
                     runtime_configuration[connector] = config
             logger.debug(f"New value of runtime_configuration: {runtime_configuration}")
 
-            celery_app.send_task(
-                "start_analyzers",
-                kwargs=dict(
-                    job_id=job.pk,
-                    analyzers_to_execute=job.analyzers_to_execute,
-                    runtime_configuration=runtime_configuration,
-                ),
-            )
+            if playbook_scan:
+                celery_app.send_task(
+                    "start_playbooks",
+                    kwargs=dict(
+                        job_id=job.pk,
+                    ),
+                )
+
+            else:
+                celery_app.send_task(
+                    "start_analyzers",
+                    kwargs=dict(
+                        job_id=job.pk,
+                        analyzers_to_execute=job.analyzers_to_execute,
+                        runtime_configuration=runtime_configuration,
+                    ),
+                )
+
+    if playbook_scan:
+        ser = PlaybookAnalysisResponseSerializer(
+            data=[
+                {
+                    "status": "accepted",
+                    "job_id": job.pk,
+                    "warnings": warnings,
```

Comment:
this list (warnings) is empty. we should avoid to rewrite this code.

Just declare:
```
                {
                    "status": "accepted",
                    "job_id": job.pk,
                    "warnings": serialized_data[index]["warnings"],
                    "analyzers_running": job.analyzers_to_execute,
                    "connectors_running": job.connectors_to_execute,
                }
```
before the if/else and then add the `playbooks_running` field only for playbooks

Reply:
on it